### PR TITLE
WIP: Rename codeGen to cg

### PR DESCRIPTION
**Pull request by pkrc267**
This patch makes naming of CodeGenerator variable consistent by replacing all occurances of 

> codeGen

 with 

> cg

 which is more popular.

Issue: #5594
Signed-off-by: Prakhar Yadav <pkrc267@gmail.com>

### CODE REVIEW

**Review by 0xdaryl — CHANGES_REQUESTED** [review]
I suspect your editor (or some other means) is reformatting the source code on the files you changed.  Please try and keep as much of the original source code formatting in place.

**Comment by fjeremic** [comment]
@pkrc267 welcome to the project. Thanks for your contribution. There seems to be quite a bit of whitespace changes in this PR which makes it very difficult to properly review. It looks like you make have used some sort of auto-formatting tool on the files changed which changed many more lines. Could I suggest we remove the whitespace changes and formatting changes so we can isolate the "meat" of this change, which is to rename "codeGen" to "cg"?

**Comment by pkrc267** [comment]
@fjeremic  Alright. I'll go ahead and create a new brach. This time make changes that only takes care of renaming. I used VS Code and maybe that caused the unwanted changes. I'll do it manually this time.

**Comment by fjeremic** [comment]
> @fjeremic Alright. I'll go ahead and create a new brach. This time make changes that only takes care of renaming. I used VS Code and maybe that caused the unwanted changes. I'll do it manually this time.

Thanks. No need to create a new branch and new PR. You can just force push to your existing `codeGen-renaming` branch and this PR will automatically get updated.

**Comment by pkrc267** [comment]
> > @fjeremic Alright. I'll go ahead and create a new brach. This time make changes that only takes care of renaming. I used VS Code and maybe that caused the unwanted changes. I'll do it manually this time.
> 
> Thanks. No need to create a new branch and new PR. You can just force push to your existing `codeGen-renaming` branch and this PR will automatically get updated.

😀 okay. Thanks. I'll do that.

File: compiler/aarch64/codegen/ARM64SystemLinkage.cpp [hunk 1]
```diff
@@ -501,18 +501,18 @@ TR::ARM64SystemLinkage::createPrologue(TR::Instruction *cursor)
 void
 TR::ARM64SystemLinkage::createPrologue(TR::Instruction *cursor, List<TR::ParameterSymbol> &parmList)
    {
-   TR::CodeGenerator *codeGen = cg();
-   TR::Machine *machine = codeGen->machine();
+   TR::CodeGenerator *cg = cg();
```
Comment (0xdaryl) [inline compiler/aarch64/codegen/ARM64SystemLinkage.cpp:504]: This will likely be a problem with the name shadowing.  You might need to say `this->cg()`.

**Review by 0xdaryl — COMMENTED** [review]
I'm going to run CI just to see if there are build issues.

**Comment by 0xdaryl** [comment]
@genie-omr build all

**Comment by 0xdaryl** [comment]
You may need to resolve the merge conflicts before CI can run.

**Comment by pkrc267** [comment]
How do I resolve the conflict. The conflict is in files: `compiler/codegen/Relocation.cpp` & `compiler/codegen/Relocation.hpp`
Please help @0xdaryl

**Comment by 0xdaryl** [comment]
> How do I resolve the conflict.

I suggest you rebase your commit on top of the latest OMR `master` branch.  When you do so, git will tell you that you have a merge conflict on those two files and abort the rebase so that you can make the necessary changes (git will provide instructions on-screen for how to proceed).

**Comment by 0xdaryl** [comment]
If you have never done this before, perhaps this resource will be helpful:  https://github.com/eclipse/omr/blob/master/doc/GitCrashCourse.md

**Comment by pkrc267** [comment]
@0xdaryl I'm getting 3 errors in Build. Please help me fix it.

**Comment by 0xdaryl** [comment]
Your build failures are occurring because you deleted Relocation.cpp/Relocation.hpp.

Something went awry with your rebase as you now have 18 commits.  You should probably only have 1 commit if your rebase went cleanly.  I'm not sure what methodology you're following to rebase, but I suggest you try again starting with your lone commit in this PR.  Did you have a look at the "rebasing" section in our Git Crash Course document?

**Comment by pkrc267** [comment]
Oh yes I looked at it and thought I should have resolved the issue with those 2 files manually. (Instead deleted them. Mistake. Learned from it)

The 18 commits however is a separate mistake that I did. I was trying to update my forked repository(pkrc267/omr) and local repository with the latest **omr** repo. And somehow I made commit to the eclipse/omr. Which wasn't my intention.
Sorry for the mess 🤐

**Comment by pkrc267** [comment]
@0xdaryl I think I have messed this PR real bad. Can we retry this and start fresh? Is there a way to remove this PR so that I can start a new one, such that we can have just 1 commit that solves the issue?

**Comment by 0xdaryl** [comment]
There may be cleaner ways to do it, but I suppose you could close this PR, create a new branch based on the most up-to-date OMR `master` branch, cherry-pick your rename commit onto it (i.e., `git cherry-pick <your commit SHA>`), and then create a new PR from that new branch.

Given the nature of these changes, I expect that you should only have one commit to contribute.

**Comment by 0xdaryl** [comment]
Hi @pkrc267...  Are you still planning on making progress on this one, or should it be closed while you sort out the rebasing issues?

**Comment by pkrc267** [comment]
I think it's better to close this PR. I wish to open a new PR in which I'll make the required changes in clean manner.

**Comment by 0xdaryl** [comment]
Thanks.  I will close this now.  Please reach out again if you need assistance.

**Pull request closed without merging**